src/targets/gpu/include/migraphx/gpu/prefix_scan_sum.hpp

```diff
@@ -54,7 +54,7 @@ struct hip_prefix_scan_sum : oper<hip_prefix_scan_sum>
 
     shape compute_shape(const std::vector<shape>& inputs) const
     {
-        std::vector<shape> in_shapes{inputs};
+        std::vector<shape> in_shapes(inputs);
```

Comment:
is this necessary change now that constructor is marked as explicit ?

Reply:
Changed it back to braced construction